New Install Crashing on Android - 4.16.0 - Solved 4.16.0-patch1

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Game Version

4.16

### Describe the bug

Auto save is still crashing on new Android installs. Just tried fresh installs on a Pixel 6, Thinkphone and One Plus 9 Pro. Neither will progress to next turn.

Crash output:
**Platform:** Android
**Version:** 4.16.0 (Build 1,113)
**Rulesets:** [Civ V - Vanilla, Civ V - Gods & Kings]
**Last Screen:** `com.unciv.ui.crashhandling.CrashScreen`

--------------------------------

Device Model: ThinkPhone by motorola
API Level: 34
System Memory: 7285 MB
    Available (used by Kernel): 2179 MB
    System Low Memory state: false
    Java heap limit: 512 MB
    Java heap free: 87 MB

--------------------------------


**Message:**
```
com.badlogic.gdx.utils.GdxRuntimeException: Error writing file: SaveFiles/Autosave (External)
	at com.badlogic.gdx.files.FileHandle.writeString(FileHandle.java:386)
	at com.unciv.logic.files.UncivFiles.saveGame(UncivFiles.kt:173)
	at com.unciv.logic.files.UncivFiles.saveGame(UncivFiles.kt:162)
	at com.unciv.logic.files.UncivFiles.saveGame$default(UncivFiles.kt:160)
	at com.unciv.logic.files.Autosaves.autoSave(UncivFiles.kt:477)
	at com.unciv.logic.files.Autosaves$requestAutoSaveUnCloned$job$1.invokeSuspend(UncivFiles.kt:466)
	at com.unciv.logic.files.Autosaves$requestAutoSaveUnCloned$job$1.invoke(Unknown Source:8)
	at com.unciv.logic.files.Autosaves$requestAutoSaveUnCloned$job$1.invoke(Unknown Source:4)
	at com.unciv.utils.ConcurrencyKt$launchCrashHandling$1.invokeSuspend(Concurrency.kt:89)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at com.unciv.utils.Dispatchers$CrashHandlingDispatcher$dispatch$1.invoke(Concurrency.kt:190)
	at com.unciv.utils.Dispatchers$CrashHandlingDispatcher$dispatch$1.invoke(Concurrency.kt:190)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$wrapCrashHandling$1.invoke(CrashHandlingExtensions.kt:17)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$wrapCrashHandlingUnit$1.invoke(CrashHandlingExtensions.kt:33)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$wrapCrashHandlingUnit$1.invoke(CrashHandlingExtensions.kt:33)
	at com.unciv.utils.Dispatchers$CrashHandlingDispatcher.dispatch$lambda$0(Concurrency.kt:190)
	at com.unciv.utils.Dispatchers$CrashHandlingDispatcher.$r8$lambda$L3K1aau81hhVdGYYei_E7yWI2wU(Unknown Source:0)
	at com.unciv.utils.Dispatchers$CrashHandlingDispatcher$$ExternalSyntheticLambda0.run(D8$$SyntheticClass:0)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
	at java.lang.Thread.run(Thread.java:1012)
Caused by: com.badlogic.gdx.utils.GdxRuntimeException: Cannot open a stream to a directory: SaveFiles/Autosave (External)
	at com.badlogic.gdx.files.FileHandle.writer(FileHandle.java:362)
	at com.badlogic.gdx.files.FileHandle.writeString(FileHandle.java:383)
	... 21 more
Caused by: java.io.FileNotFoundException: /storage/emulated/0/Android/data/com.unciv.app/files/SaveFiles/Autosave: open failed: EISDIR (Is a directory)
	at libcore.io.IoBridge.open(IoBridge.java:574)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:259)
	at com.badlogic.gdx.files.FileHandle.writer(FileHandle.java:355)
	... 22 more
Caused by: android.system.ErrnoException: open failed: EISDIR (Is a directory)
	at libcore.io.Linux.open(Native Method)
	at libcore.io.ForwardingOs.open(ForwardingOs.java:563)
	at libcore.io.BlockGuardOs.open(BlockGuardOs.java:274)
	at libcore.io.ForwardingOs.open(ForwardingOs.java:563)
	at android.app.ActivityThread$AndroidOs.open(ActivityThread.java:8275)
	at libcore.io.IoBridge.open(IoBridge.java:560)
	... 24 more

```
**Save Mods:**
```
[Civ V - Gods & Kings]
```

**Save Data:**
<details><summary>Show Saved Game</summary>

```
H4sIAAAAAAAA/+19XZPbxpLlX1H0y+5GiDeIb8Bvkjy68q5lay356mHCD+gmpMaYTfaCbNk9Dv33BUgkQIKVWXWqCm3fmRt22BLJQp76QFZV5snMP66uy6b9ty43u6tv/riqNjfl3f1dtdm3f/33P65utg+b/Wr72+bqm+T51f12V+/r7ab75e9X3yzC6PnVY/v/9Ovzq919+dumWv28qbumy6/PzxqnF43D8NA2jC3aBsGxrUXTRZgeMQe5tnWuaJ0fWgc2bYPEfrQWQXxsHOphZxeto6XDWB/nyWC4LuUuwqPgTNH2l/azm/pLva7/s+ya9Out/vKq/fO+ebjpP/zj+Lsfyrvq6purl+NqfX5V7e7b35Sfq7dl99/m+OPq9/v1tqlWP1Wfj0jazz5v16tq8+JzdVjku333/Zvy/r7eVLv2o6L7SVOV+3fV9n5ddb+539btO/B62/xQ/b7/e/fd36tN1ZTrV12HD7KuDh8/6z9vh3e5/Pq1G4N1fVNXPfT/91Dt9mf4mmpdd8jOP3yoN7uzT/bVzW0npPv/7qdqV5XNzW21akfp6sXnpr55WO8fmuqqG8Yv9U3bo8eT5oeZ0Qzl++3m821Zm4/j1fV2v9/e/VR/vt0PE5z1y7Kb4v32/vvq0/hd0L9p6dennoJ1+Vg1Hx7vu36+ebgrN1fTeblvtr/Xd/X+sXvKh8em++UP203V/vDddv3YgqrL8aMPt9WzNw/dGNInL8vrx3U3LMMHTbuMd+vyy0mz72/L3fBXl8Ww25f73Zu6G7UD4KCbv/iH5avlu+Xflx+Wr8P8TfExeLG88rNy9odxOryP+8fpKrq6fqjX+5ftf1b15nP3u6t35bq8aR/Zrq+TH//fh+qh6r5+u908dHtK94N6867ZtjO92108d/jZN0k3i+2jNtXNvlp92L4q7+t9uX7fDsNDNwlts+p04Q4rtlUM/WLtu/n+sNCOGvtTu246yK/qL+0zhjltMbU/uFrFwU1YVcmiiMtoEZfVcpEXWdXqsDQqojJYFvF199vdj007ZZt2GR5BXX3ToXl+td7elMqNJmpFb47v3Iv9vl6X/2P3rF3Czf6wKO8f1kOzT9vtigCHJ6uj1wj1up+V6eMnn8QXn4TDJ9nFb5KL3yQXT86GT9oZ/G3b/NpOyjma8Sm/dL85LqFX3dt5GP3nWnV0Mh2YXm+3N612SXxql/al/rcvVfNjt50xa+FE19BC/PHT9+Vun394aA5bXfB8efpPO2S72+3DevXjfbV517V+fFff/NrBOD7x2KVX/bt7WM4qFfa6bNoB/Ee5afflL11rUkWk7ZVajlqt19V+X6qU3PEXZzru+NGg4qp2W7jeNt38/dzMofOWnc7Lz3Xe8mN4rvN27ahvbqaDHV8M9kE3fjeoosMINof18vnqm7h/4FR5buq7ct0u0t11uVm1oJ6f69O+0YctNesaDU/95bBBtmukn8N25bXvw74dnN3h/fjvrIppnZEmXgb5dVHdLFar5adFXOU3izyLq8UyWQXJ9Sq7Dj6FkCbuzyjJqIhHibwGjjQauL/8hCmpyf68E466dfqLkPtFMn3G8EE0fMCp3uGZlpp3HAtU8Sb/Urw+FO/J6ZLVxGenyxNdS/r32HBGzRtPNW/gTfO+rTei3tUq2f4B/6wqNn0CFTuuul7JJkUcL4swXqTLYLmIk6BcFFHxabFcJcsiz6IouEkBJdsbN7pzJenYN9vNdv2wfhCVbCAr2ZPHHv9+NIMMR9PgeMINSF3S76PJ35PJ76fPixntOra3VK6n4/6vc621en213u4qQwVrok7ND7b/zIr13Xbfzvijg2alJ/xFVOv7dvJ1evX4m7bLT3BuHZdNr1Wj1TIJonLZqtGiWMRpUCyKT1W2KMq4VbOfgjCGtOqiODGv0dH1VKi9/SBYjgbl4yf59LRaTH9RcL/Ips8YPhjEZNz5dRRre349HZB/6Vj/R9jLE6ubzp37/PpfQ3M+dqqqelc1u3Y9rw9TdfW6aXu0Wnemh+EXP2/qdqQ6v0r7g+/LzWr3a6vzbvdX+jdnmKR/Dl/KP5fN3tQMr1w9L7rf9qvHfoXIs/9z8695/6vN+6nWeKIz1ceyaeptozlV0a+e5Fx1mOP+RNUen5bZKl4u8ptVe0/9dB0vruNVsCiXWbHKk09lvkLcMj1lIUjHA1Uvzfoo1TMZhjsmiaAzUBgPIs9+n05+n05+n02+5yyA4/PZA5RSVbzZ7rp+nO4l7VLcPjQ33aj87+q3at2Y6JF+/PyfvLyqlL/kyUtz2KJb7z/RFfe/xtnrT9WwT+FxOU59r2I/Ba0ajW6SRbkKskV8nYeLfJUu2ytZfB2ubpI0vFlB/pbeXXxyaSV59t6Wnnm1GC6j5NxOpx8Md09qMn6QMR/k0w9S1tsySLW8rdJI/Oui+mdfVE+voVb68y+jMf90B4rjTWScSo8XkmDp833YtMq3/fvH7WbV9vLo5mmbPHv98B91N6j/bW8sJ0tHsTw6FfnQNNVm/+7AyTvjHfbfdEu9XT/N/kPdrYYgi6M4D7L2/3GrjVf1p08HCN2aetfU7SvSNv3cLpzvjgbX6FP66TpapFUcLOL0U7S4DsJwcROWcRkXeXJdLvvfvyub9r/7w/T90XME+23+tj1nbY474whPwSJsl3v/7y9Dd7sf7I7G6XbB7+sWbdv4+Aa1C+j51bfbu1Z7Hva89i91u5Dv2r91Pzp0uBu526NP4DgK7XBcfbNst9VWD3Rmna6f3Rm/2w/flvcd+Lvy/rw3N8cd6GO9v/1H+9HhHbmK/xakf1s++5+Hs8qz4HkQRP+rldo2fl//5+F9uK2OZM44GnfqsvncjXBTruruuNFxh3+rV/t2stMDc7k6bHTHSUqLLOl2xP1xkD7cNlX1rFMZ7TvYUcj7Xfz7tnuHgb4ud9WHqmnKusP34011HFPl5woOdM8x+2rcJOjZ2mYNkIcDj/0d+inSvYH7Yd6kZ90jeGAZASQDeTgGHV8vC2zB/I6NJDxNFksBadJTiyLzJscWQAOCBTQJYRkBLCOAZCAPx6Dj79YCe7mG8bRQK0ALZLbg5WaxpBHosEKM8Enrg3di8ybHFkADgqVq8mrbHiW4riMyQrgfISwjgGUEkAzk4Rh0XA8tMEU0vs3QtoOucPwlWsBvKrIi4NcGQR7DnY1hjRHzk8a8mjG8Ccb44utjNhJzWMcWqgYaWICMGJYR8TLEFY7ICGEZISwjgGUEkAzk4Rh0XGcvMKU9aj788AkfOCDtCp8fcIWzgLUasuqA3yawdk3gziawdk1g7Zr066KzkA0Oz9f17hYwRiT8chSlIu9Igr9Wg3vCGNaxhaqBBpaqybt1+//OoH5DRqDOEMsMRXo+Ad9WBwPw/viw11VncT1a1l5v2z/sO0sZuO0AwxDDwxDBMiJVt8F1F/JSRcWJ9CyAZQSQDOThGHR8C11ge+i4EeH3JqAFbppZwLYZ4VQqv1ZAC1z/L/hNhlNgiHpE9SKCPOU7K8oInRRCCk9jCl8uUnizS+HNLuVfK9Wu0nlElDKT89H8eNtuJPhwIhpkYBUYNzm2ABosUl6GOAOIjASWkcAyYlhGDMuI4PmIYBkhLCNkZTAdp204c1EPAdQzpEvYgOFHigV2phg3ZvxaD1sOoM0fNrNamHJge5FwaZLVA3DNGvZD9+1Ntacqd4XwUtdO9oP6wJYw1u3m3c3gAwOlVUFlILOW8RMtynC7HWfwgSGDDwwZ7PDIYJNcBpsvM/i2leGqceBCGsM6tlA10MACZGSwjJSXIZ5dERkJ3I+kl6G0RbzYb9drC1MEADlm+8idFCJYRgTLCGEZISwjgGUE0HpAHo4ts+Gshp5v4BMkpChg2w/uS1nAzhTcG76A3eGCeVZWFPAdd+Hj+g5vz5BJhT+ZcLsFshehmxCCPOc7+/em3O3W5WZ1fgK8NDbnKiPLq3K/XyMHQQpTNzyIcjAmx6o322ZXdU3psxd3Ha34ABdEBizcHD5r5fBZK4fPWjl81srhs1YO2+xz2DSQ41tBn0ehML3lXGYXPj6hOF9dL8tN+89O7VT53w+bz+sKOcnkPEpx8FQNNKdOQEYGy0h5GeKpE5GRwDISVgZ3WophGTErg9GmiqTfEUlVrqq3ZdNuf+aLKoJ7HcK9DmEZwdhHR8ubKUykS9hSxM/UC+xQPZ5McQMqbKOFTr8Ruj8IbCX5ZAqTKKDTL8wcEPw1slKEDTaQBRG2XgnHP3mTks66FEv28rFTWCcZ5X+53FWRPdyTyB6/H+NDobcwwtiAu8qQIMpIkfInehhjpB4/nEhSwIfrAj5cF7xGML7kFPBxu+B13bfVrjqkJzcQOWGzvN9vIft6AR/IC/hAXsA+xwK2MhYWu2yfGi1YGk6B4tjfP0P5CJ1OULURxwOSkgtSlHeadgOq7+6b7ZfqmBX16sXmEFD57BCwyhoTlm7uHXwsslGuDw2dCgjE3R2ajQSXkvBSNBeS6Zxg5+YYlxudyPUwJxGOIMTHN8SlBLiUAJMCPR5chMOlCr2IwFc9SA3zXgr5IoIzUoAWAn9VvogALYTwD1mNAC0EUpesEIEWgq9ZVvtAC8H6K2+AQAvhrC5v5Z5O94Zqm1VOZs29HPLp2IPoTjptgXuE1UDC96QhTa65ewEHBAUmLvm3yucUxurp4I3TuhEEXBR9E8DhMEgB2vRNgBvOIEXVxvhGGIy194yCLN7U6y5bCX9F1qE14ymi+BkahhatbmEDbsu+CXRwWVocduh4B6i1vo2yifbeichZ4nIKXEyBS8lxKTlJ8cBwQ7BmeO9SXEqKS0lwKQkuJcalxLwU3V0QTgEBSQlxKSEuRVII8n3PVAr0eBC+xW11IVxX5Rsfomh5f6B840NulTy/Tr7x4bEUQAshcFNWDUALi/gZittBb3w4WRhoIbBq5K0IaDF4R9BNFbhajPce+ITg614JicaoXhCMwbNmwgNjdbKQVcwO57nVe/ju2ViLvX3cXb2u92XzeKzz8AflTnw7NAyPyXyDdqHf9R++re62DaUyVo/GcSGx3/1yUnXmYd+lBdz2uWC7xLDN1aQg9/byw6/cPKiWMDN+wWWo3zANoT2fzmopo17yAeh8F2k+k4YOk5PHMaDAaT/vk0r9o1igO7YQkSZftqYBRbdllyAal6zauM2opAERdM6HfntINzpd+Znn277AJ5LHzeYuDJC2AvzsOCpzqBEd/pFduD/9Q5n+Ags5AclxudsOZwYk2eYS72GBd7DApeR4Z3JcSoZLyXApKS4lxaUkuJSEl6LzFsO5k5QtDum4uz/yJgE4BZ6yBXdIiUD+wmJUIxM1fn8v2mjNjZ2C0tGZHOA0u8hYpQojRDgbdR0cAgujywK3wloQBhYCY0A2XCA7L0/Ilg0XiHGED3CUDRd4MgughZA5S9bGQAshnFjeV4AWQqiOvEMCLQRWpLzXI7mRBZeo5uCCKC+bJOF8lnD2+AedL+dxAuM5xYeTridDEJ+Jlt2OhHy3fm7PfI5IHSapDY5A9fapSKABeswguNhlH+9AIjhqkfulkMXF8HbMBz3rhJqmNGSFZtggKFm+EY9vYnp4X65Vqa0gxLb+dd1QIvYBiwCtwe3jQFkK8APVqBIB3mYguP60cqBGdNMxV01jOSLFi93fgiJDg5e2L8juH5JsC4MJIiewkLO0kLPE5RS4mAKXkuNSclxKhkvJcCkpLiXlpXDKJxmkOBjZElxujPcu5qUYGEzwUh1Idyy0gqAUzEwNpyYOuE6Lg1zVevFs4gCG3sLIsxCsPIyUcQfzEymD76JjXAhq/0Ca4GH4Cz4OXzaA4Hmv4NRakJEFzjFqkesMTqgm5BGR90CghRCyKe/mSDkZgYKtOZrgByDMzAIXoZIsCZpjI2Jmga7xmEeeORtDAv1aU/jKEzoEUhscgWoclXUAeECmjwANE3zmVd0IxU73Sj4jGmPR4WBMyBXvb6vqHvMMGdoCxsoEjtYLoTMOxhWKn/NjYRIC+Mwi3wNid3rFo9qqITyF82rRQUToGPhBatSQphHxwWW6X5JrZTcxV80mdhNoS7VwUi8CuhGaJs2/0J9n3eirEHdPX55/+fzqh0Od4mlH++vlRFseYvDbT7blimiC46MshkRS4PisTMFaW7OQSLOQZFtYsxA5gYWcpSBHY82yyHKCtClwKTkuJcelZPjUZLiUFJeS8lJ01iynI0+ifLfACGJJm8n2LVULA/sWXJAHGlhJT8j2LejEavH2YlKgx4PwLWxxC8EYp7FEubEicQqnRSzaAj++jFlXUAMUng4GsnLxGS1lCxSeQR5ogRfrWcDVeoQss/K+BbQQ0nfJOzBS1lOIqdccJxB1ZFM3GS6cLDFJNEc9hDw42HicUmmQcckl/Si0iSAqm69Gyj9eb/lSS0EIuLxc09zfMhbAzgUYqQaQU2/iQ8O4hmypGQEl0Te7//DDbNM7Js+Ho9mKT77K2KZsoIPEJgi6ZHGTpthj9zwY/djuZVNquzJCKYIAnJzrbbrrYPXjacumltCBKz3fkKuOWyC8Yqoh1BUUoYV+kaeReah2C0F4V0L6CK1JzZMpcIhYcTQ6ScEyBrCsTXsqjosH054FlWURkKUCoIZ6HZLeyME4ZdRj4iOMZxwuPz1npprZ5yxSLEUCXI27AKrcHFrICS3kBBZyAgs5S0GOxhJrkfkZaVPgUnJcSo5LyfCpyXApKS4l5aXoLLFwMQFIyqhL3JP0InIlpSAbXfGCrpAUQSPojK7Ii4pJgR4PwrcwGS9wx5IFvXOBk75t3Ka4e9YiddkCP3xaJOleCFm6ZSMrXjAJaCHUDpUVOdDCoswWXMtLqE4gb65ACymlqOakgKgfIeWL5tyDmHKF0DjNKQ4x5RrQ2JgzqZMHyMKMuRCspZwUU3OWKk6MOcP/WO7qHXCEp6sO0tGEHxxZCjSNQt1ybrfj63oCluShnijI1jcnlPJFnjxZYofCU2AXDO08lBveB4XQpl8OAaOU5PDJbLFeU76FS7OUb8NIHTcn4ds50r5NpFtZOPmcDqxR+Ocuk4rZAnKwDPOxFqamVyHAA8IABdEKPn1IKFbERDrmyjocS54rZTVmugcYzYSER1oDpqoRjsjagHmw6bpacMcgyLn78tOhL+cq81W9f3x20/75kCmJ6Wfss5/QwZnsRVgCf2D19dYl2y1L21uAq4+/M6mZK0hrDVeBZDcCEws4cs8bjNxIAsnIQk4ITkl4WbOXjN6gq1drD5e6bpGZFxqWpWcMS+XaBCmCBb4mCrzvOS4lx6Vk4LrLOEO74+su6VLZ9o7nOYekJLwUne0dLkkMSZE0k2xpR6RIekm2tAO7Yni5qhaj9pmx2L2gklhzvelIgCM9OhuccrrCvhQL/voCj3OxyDVBBnw4DwbmJhCSjWoM+HjtFbzAC+YmwEuGZribIEPdBDnuJshRN4FQ/FfexYEWUr0uzQEGcRQImdI1RzXEUSCki9OcURFHgRD4rTlxI46Cgf8L3y6Q3AaC5VtjnPCUGgXat5ENm+6ZDh4J0lZerh7908D8z4QBcRqk/VrD5gdwKKSjQ8Eph785XxRKGxlQGVaj6u6mVk6hHKw8EXNRvIVqRYYWUMoB7Bp677E3DhZ2SujzJL0xsc0PSYnmYEzzwWOm+AXI85C8eSoG+A5OSN4v1g939ebhTqF8IkewmHdAOt0auvOGCoj2qtUi76lUqtHddDpUcfBVLnesJIFtce6eEyHjotagPr+3Qe05GeLqPXgUpLD+p6CxD8zUeTnrEmeWbURGR9Xl68PDZtWU2re/tzoyO4aFTyR1x5T6xpS4Y0oIk4XnReOCZ9wrkKjITlRkISo0E2WRax+CEQgwHLwqEIalZwxLHENhAEHtQ8GpoGopZt6NXHynUXUpKT45jgHP1wlJOVFd9sZnSV3JkQ14PVxIiqCZdN4VpxER1JTO3wL0TtBCusgGxPqJSRkGz9XwBS4PyWmtcZrALi7s1IOHpFkkMFoIGYw0ThPkpC4kHNc4TfACdXgVPMw1M9wjUacJ5JrJYNdMjrtmctQ1U+CumQJ1zdBpw+KEgrz3QqFXzVkMsZQLZWs0B0/EOTOkp4VP0ohzZshIAuwblCPFJb4CJ/fTTQhx1wweCdhagsQ2QJsuspwzZ5cMaSlnK68MEfXSZLiXJgMWHeCbyeD13zcBinhxQsFEOTIelxTUQq1CWSjDMoQsila+AqG4gKHlmDK9gWYa3z2Yx9tBEZazOHKE3jjENBCj5KkhO0REDMc5LBCP3xqEIyUGCdrwSOw83oyh8rWfaLrgkvI89iC0d2FbdQo6NUllsbwEClikHR5AuVSMPAM1jWTrotZys6C1MZajw8L0LjkNWXtbtg979rFsmnrbqdxp6Nqpn2Or+Oy+2d5tOyG7Tsrp31qVU/5ado9+9m25uelWqMK9MmDyQ0Qdp0O1EzO2fnyBeI8kkejjbCOyyqq2DEO3Rm+WZbY5h444YMp8Y0rdMaUjJgPtqHVEOSBJ1Ei8XUdGt5UDyFicQhhRJCCydDlFhNDC86VqZFcb2G9kc2S4SYSnfee/nCOueXy839gq4NRs4RR0fe2XAkoHPyFubpQhqP2EzFTBBQml3UqOw8qdvGnSjiQapqHxlfYY0cgOSZH2D9kziEiRNgDZM4hIkZS67AcEVtBC0tmyHxCRIqkh2Q+oaqGMU53sspA4sDsWXs0F6NY88f0hR14hJE7j+0M8DXgiPYtaLOT7A5pISQQ0vj+4xDPmYRz4lrDvD/EwCpYZ2fcHeRhz2MNY4B7GAvUw0hkCOrAuYR9jYOFjDGAfY2jhYwxhH+NQFRK+aSA+xiFjP+z9Q3yMQ0It2NeHBIAJkT9wrI5pFQnZK6b0OzIm69ty93K732/vfqq/dDeRffNQsRYKJsIKTv0GhEMM+L6vPu3PMJ4g/3zLfiV1anJEfrW9vz/kevHgnsxZD6rknsz56ZMFoX7QnF+zsiBTcrxqief0OnocYbAqG2FAosP4wt0uXqChgvgcbjiq9OQSWIf0YiJplkAfPrGqhw4w3lBWsZmnXOSDyT3ANr0HsbiAIC8E1yTw68daWZLFcSlMy33MGEIodNXFaZpbOE1JLOA0ZcRCFCgSC5xyeMnTgGSwVtaIxlPsNObNEmq5agGrGrn714aaBmfD+nNTbmqlEo59+Yik+gveusW4DR3gqu5DmOtvQsz/tmrPjRZAoBsjGaNVOtsQfW+cZsguDiPqgCn3jSlzx5T5xpS6Y0oJk4Xn1EFsIg6FtZsUQIT4eGJ+kDRZ29WQ7ByUjNt0cip6s2121U6hnkML76oKvBGHyijGEFp0gQDJn3cRgrQUIHHXvqVy0uCqapLWll2IeOyFWopo5YWkSJpVdgrilUAgKRa6UVKNslMQUS6SupOdgogUSYPJTkHTdzRU2HNG1ePi5xc0jM5viDgCMCmqfr2rymZt+uaDXbLwhC5AV2jOuhKRQyge7WmRKHSBZwqVihdqXIlwJTTMYXnGLYVciYjDEs93MlKXYVci0ESyVmhciUAT2mMRh2UBOyzpFGFx8kA0VWDhsgxgl2Vo4bIMYZdlZOGyjGCX5VBDG76HIC7LobQN7EyMXZhYYxI/QC5lFoQdh45OTArK8+jgsfJlSr5C0SEoeTlfr7fb1bP7494K9AOwsQsuzUtzkGeUrMPS1L1HWtjP7Bcmfs1zRJtOWrn+uN2sDpzaww+fva83ZS3Y0xAfaOHsAy28+kALKx9ogftAC9QHygmd2Ki/a7YblQXES3wqXyYRBG1YXcDQ/cKnfnZxRwk5qIVUVnw+BQ9YNIEHaiyytqy/1Ou63JwGeXWHH7PSZGdOUw50cUrhf191nilVhNch0nY7/uWry0BNb9C4IxNaelzg6BxeUulojoEGtiYImYMjtaD70Ll6qJX2D+AYRdAMw0oZtFgu5oK/QbLHTMKpaoQNIXSqJrHzOFOHMvS+krYSXMBJymDy5wmVighhmKZ5nNEoHykVH4YkV/hkfY8XcisPe3tBqNJ9Zl3rH6F8gg6rstH7zfY3w/HEhOauQnMLoZnzANONFdwRdX5eF0SpxUAk7mITcSAsXpkYx4R7enG8OjdwuDSy1vh0DCuHiMUZCuPqB1U4Dq1vd3UggDckzQUW74ekiWX/LyRH0ryi6RqSIqla0QwPSZF0q+hSgKRI+lJ0j0BSJPUounogKZLCE3290Psf4VIstIykZERHLqbLcCnC28+V97t07RoLdBN14tiFfEawR9wiXHchxOuauY+ZHMU+j914BmOL4pfkBwaakB8Y8TglsLdZquGg8QMDTaTsUBo/MNBEoo9r/MBAE8mko/EDIyFXxCazOIJA/ubAwt8cwP7m0MLfHML+5sjC3xzB/ubYwt8cw/7mhPc3azzBiL8Zr2NG108kRFZIQwr7fSWnlkV+LylIEQs59OSQHhycT+eQJq+sD/zGl655o2492Vp605iDw5rsc7PlKCbrnTtEPwu4fxrgvkYqAxJWP67t/mka17YNPkA79k0c3N4kFNgrQiraa2KGAmHYV20MKSf++btSrlWOcBAVkv6GT87h4AskIFiKOjW0iS8XDcLjc3eqbmVg/6axDJCjku0x5052z2UtdMXBoUywwV2VPw5BuDlyGTtY8HQwmwUsAe1ZaGoI0B4ITc3YHx6bSgGJBoLZifDUz0KnI+fDg6fOMjn9+UFW+WCknsaK6XX0ayqtIaxJkoDY+9FJrGFiDC0Se+85IYFiL6TCeZhYGze2Q2ZbEotYMgaxUCOyKKuuBSZu6P4ByvZaP7+qEebnn15FLMkU1t0n1z/S/eGabt99uqZfVFP2lo9p5Ao4oMxGlD65Ag6IUn66NJaoxXSwmQuGB099oh826TCgyQfnNH6x1Ywqd85CoBRMhvrbx4MfHdqtDel0kfJF4gjLWo4BsLCINQCtxQBai5xkC50dSDpbwwjAvQtqMTIjAI+Yg6TkuBRJg8qMALwgIiTFQgsmuJQElyJpKJkRgEg51TIeUo5HOAJJXchsAVULY+f6glEfDjdPmXVgjxYaHueBseBVLARihYZ1AB3QYW6DRcLzBZ7x3KKsM/EG4HrTGDsh4dkJYjVtjJ2QwuwEqYCThjcANBkTvcG8AaDJGG8B8waQ6MHRBIjzBhB+ApGOoNc/wPkJoQU/IYT5CZEFPyGC+QmxBT8hhvkJiQU/IYH5CakFPyGF+QlDLVFjVwVzJ2csl3AVszG/sCtfYGLUxdLZatgDLmwGiok2utzJPk0/LAbGFOaJrzAPRL/R/b3B0C7hui7Gn8yZs0AGU4YPplXgjB+wfANDDUFCgb28bwKoJYhEEOAkgoBVrR4BAZo75AvP+6CBBLTHuZigQr48him5YCjK4QEHQAww9ZoOaX5mcwcTFdQ6hhjsCpY/k9Gjpm59EJvyuYa+mYA/gftA5uA1Cvg7CKhgcdIABlGlwE0d0ARxcip7/dAwVnStlcfIqcH52wP+Eov2SOVS//N6lDh6xalX1snANfgAE8gIxsFFjxdnG8VmLs41BgjktJcySmJijQltQBzoAM8+MTWdYhEjx3CGhRqRdVulZpVOdeC41D9c+WxtL2YBFFgAWs4JaIkDGu7acwCiW7gfTvZIn8DuY0SZgIYlE0QZOpgzv51PzTp/jkmd5+3/lPW6bNUsy5KAhipxH6pEP1ROvAcVNIz3oGJseqA7GBNBbW45WFajiJ96LfkBaBQIalxLfgiNzgqysR9b3JK+lp0XkBxJC8vkBzxMFJJioTsl1SmTH5A1kfpYEoK609EhGF2FUggSHoEyuTzPnDDMgCaTKRxwMCrNi0dA0E2G6EJ52lB7O4/n+/LXSqFeF9JJVKZSWPZauSj8BOy5zQVDRYEZvDzRxAyHReKOhZC5gxF6flz+R7leV/u9KpueRWWKhVCagvSXsXzidSB351jPHjGWT4wPxK+cwHYHYnwgfuUUNqoQ4yNx2ZyIAwL4K6TaeRoOCNBEymWp4YAgedbHYCmcA4JoWcm6rOOAIFwTIoFBXJMQ55pEFlyTCOaaxDzXRHMdy50O8YkF+ySB2Scpzz7R8EIQ9knGOyRNeQ4ZTEYRyjpj1A8kd8aQy92V0YG4t4eMAPaeUT4tgewxRjgjjNESYQgDbq3z/fj9dvP5tqyVXjek1yFK7WBhuOYBIRu0J2JNaEIAQXgqIb8oZQwu3JAQ54aEs3JDQlJCc+UTCVm9aEpGMOsBQiYJ2Q3C45AC+0/IVynywW4Z6iZh9eI5egmfSIufUcgCOiCeHFDQmtshT9wF31c3mshwdDTrDYjNJatESIfnOWo5SJjdJRp6XEOYbmJEhKF+2DpKMPSq7UKiPRI6UwaLMashdOeREDaGQQ1Otxr2bb3bb5vHTuZy+Pz51Q/dg7k+JUoHULMtV1Q+5fAU947DRRpm7/qB9ui/o6m7S83HYoW4MFJ9Ueydzd2MqgMSBzLMELg2AxlGCqSD4E1Ttv9Yr508FOPB3oE+OUR6mviatUCgxBzkvVHpbmdeSu++QeyFQy9mARQSIM9rIJgTdGAxiss5AS1xQAPdaA5ARDfyk1ZxpBtpzkFGdgkiIUGDlQkAmKO2o/mESEue89OO7CXTsfSeTJWoTVDVrgSfALW92npAjRlF8QDVpRCa2spvjV5bQQZIpsyncIHeKWlf0VCWIDnSVqAhKEFyJA2vISjh/jtITIG/cJLGlQlKeE5+SEqG90XSeDIdCZEiaSiZcmTqixEoR4DQ2FlodKLg3Hd3QXUYETlelteP6/ZpKisNo4kfPpfcTdQyOtVfH5xTlQfqTr+q259zGbwsy2/PM3O+1pbbKj8luvo4wh7HCuCpWV0e6dpqOSN6UhTE0TKo4QSTogonIk9MFhEPiIgmhdC0ErINiWdvYwjEoUI4GClsFSMOFcLUysiyaCyFGFNuTK2cjLng4QhjagkFgHUcKosjJcTVGmNj4SMyxNUaHTnwkR/iahEZE+JqReRwM5cT844+DYsK4Wol5NeEr75IXqCU/L2AlJT1QGs4U6pXHK7Cg3C1ct7fb5rDJ+85CZ5y+BTEoTB23zPWOLzMB1gEyZxzRGVX/FTFkxKxoLwkxtaL83iU1DAfY0ccLPMDODyec0379NbAcRRYXH3XvaADEiTRNuHkdCP3CHI6U8E1IJ9F/Psgg3Mhn0WkMrwQuyJWZYF4Jtyzdoty8/70z50reVFE+4/HUfRERTtnWQ3fPas2N+Xd/eHj51d39brel83jz5t636G4eWia9pu3Q8NWfdWr42Hlrv/wbXW3bepDD/9gR+TYCeHbX7ow6Ltujl82D/uOltA+9HO9KdfEPDjpTvvl5YeXHIaJdGTQ56LbDWUjPS4QoHgUh2dKSMApVxzNjmIWsDhMUEswj7fDqrwaAJV0xluJierK5iC2RXSXsnMIgaPvkvEp4m+XRjruw2317M2D/LLNTcSL6CZuT6VhjxBG/fNO5Yt4G4Yp0YuwTcnjzKlrPh5bRFYf09lR8N1mqkRFY2SMTV1ib1524E+HUT0/PbT3lcdnN+2fD6w9ZsQNmYMs3Y5GB8h45W+5ZCrw4Ohre8acAthd0cc7Dedn1PbCgYRoEBnNinUgFwq+By31zoE4qK5lgxIHx+oYhtwsBgaUEGtMGwE0Ige0ahtxprX1rmqQKa7t4CxYI8IKAAp9AvJ790vN7n7E34xVV7+TL+e4+Z3LNhzyYM41EFgAWs4JaIkDGnihxoDU1nBMaC4I1eeTg0Rlrv3LvGqlgY9pakhbgjRLCacVYzAxmytX/mvCz602yxzQKJA3GfacpiVUIhgsNo9A2jw09MnJfQ10sQeSDtUQKqEeSqpRQ6hEVo2k8GRCJSJF0nAyoRKRIik3mVBp6rxU8AElDSbzK1UtzCupjYrIyWclKBpzLJLmkYmYmFxj7mOkHhzQIyYoJtXeJBIbJzd0+oq5pC9CdQdelpv2n07u6dV9fJgPh75wsIR77S+MAh3A4HQAFWNlrlRnGA5264feIjfFZREYQDxOe17pafrDk2X9XXOYSZeITP7K74v7iXC9hIqcrnBI106cL17yW/aKXGkCMtXKBpxRxpkFayoilHohtBKb1CktGPFLvWQiJOYpwm/NyLhrfGoi5inCb83J/g2eGbEKlwX5NdADMEZnJZ4pfqAH+axCYRUdzxTiswpJP3Q8U4jPOvhgzeUQjx2oWUv32qm32S5fH8RwTXrPvrmUlMgG1i7n/hEI4TUjxoWLU79/Smbk0gdqKOY8PwUAl/c8IaerDhn3sGxcAK+tF4Dk4loSCQyBpObgPZ+YQxFC3hEGYwWy5rsCaRZRvitHzbWMYBOy4VknDjROwscV8rLsy0AGlXRQ/aVetwtF6wnBSHCBxIELTh0h76vu3W0UrpBxOW8nn3w1ngNTR084V/c+lk1Tb7HutTPZbO+23aN33bNP//bvVy/u7m/r63r70K0FngiIZOv0oXRUbyjC449Z1F6Q0KvghYvYX45c+dF6OnfMayMZG8aL5+R6C3uI+13F+DB0Pnjft0NWXmxrHGpbT48Mfi4adcwfRWRAc6XtjPlzmgxIdez0CAi4/IaU19tyEzcD5E4kluEDyTaFd3gSKPyxRgpDhsONy3JqRU7neAe0e2HNM4/GdEEFzU2mW49RN6dkqMcKDoPSTNNFdflyXe1sCwYb9cm+jhlhdongMoIIMIYZiAA32ALfnAlAmQ6ZVnm1VIxMDtG/SLLPmEyWfvOcWg1U6pet7290Jtaf9/X6y58xPtkTLyRmNKyJxFaddiAZD3k9fBGfx3QjjpimXqr35XqPMORHZ5crEDNOshZIYGSENmIpC6XutITjWcr2GiTZQwjHFtm5FiGxWlR7lXMHe66Ln9jVsYOzYI0JKwAomhNQZAEoFAAZaWij9LmqcwcAMpgXZGAxbkt0ItVuBUjoQH22FUrUZ0+v18CKVuHRs6Khrmd2ojJelCbTDLTa1E5GdJy1jGYHRInFmEvaU8NWhuRISlHDSIYmVtJ1snsfkzOqqzOt964qm7Xq1q/hJEOyJb2k4SSbrq5L/lxT7uvduvyiosgsCvVogETPYlaMuReMgl4z4yGKGLMTjD5s/4JqNKb3ioBTGbCt2T+1wT3Flhhgg0tz2088T5OWdLDM2/aBBqX0RqfvkQulV1DvFtxzvhtwYPki1HTRY9HwP2MGgyfonrCvOfC18WzEKpXP8JxTL9l+Tcu38lRwiDV5HGZ7/jmTnvrjbTsEyPWNjBbGOGDiN2KhEULPfRG/EY7mYPwCuEAH3s/SjBkzMroVzBjaBpdmxJ+h6yM1Zvjoq/WojSDUpm0PN9WETJ2mFmEDFjpSNTsl868H+cQ5R9I6Z2QuN75JEOccYbbn5KIAT/UYs70gNxF6wcGo7cQ5Ry5SxDmHtHTA+zJ1nHOLciYQt33MXQXf5iFu++iIh60TELeddCHEbU94poWGYw4x2VOWWqIhlUO5mrOef3PO0m7Ka8AskfO0Hw+sov7xSKLngidGUe0CCEHB0rg0HHF/RaStwjIZIzXCYOs1GMY7na8UF5n+3YKEpaLmLll/rTtFZdrPO3Vf33DEo7dls7u1YJh7ZcvPMLuUz9gru36gFFvqJz1dGaFwu+gECcnk3qquSKlO7AmvpoTttZee8PMlA7JO2GmAxk9Znf5pLjmxE9JdfrnTSa9XnYF5i0hK2M3KJxU9oZ3a4/RikXMusCVaGgzbOr00htlTmmm+yogPKrtQ90QGpDqwg28QcE0Ih8uICccJxIFciwiI9SYrMnAJkOrpGkBekjhfEDPpOzYXDCEOFUSXCTlzfJYXhnnCX+tt2Nr2XY+m2WqppS1ngbrmkL3ZbtmpdkWuCF7CW4kwQKbqNID9v05E24S3tvngE1OnnPMrM7hVe5xP3KbPR3GbZycxBKp6oBFPTUjgctm+sERn9nQWnWjYZ0/4qkaIUGB3GoVaZ0xOeLeVlqprzEVmhFrxg1WNEKG+sipL6UDZRsRdUeklo070LBXEsTEgtRaaWAiNUaEWFGkIUCQA8kD2jQiS36yzA5E6NTqa+6NW+7kSjhxrYNjVxmPlE9hhW/oYNrU5fvpML8VvWM9K4fjyEjcbnE8WT+6KJ/e6vgYCt/36IjI3gwi2OI1MbwdMqXKp+Qs5H8nfDiCT+d8HyMEm7TkaYrnUBMIgbTOyP9x4JhTENRESs4Vg0TJBCINkKAnY2AjK3wNBO1haQFp6hXTBpBy/5WwjhXpClUG7J48D6TN/YgdzYcW6dy/nu8em8zHj0AOrKPMOQtp1ZM47BoInlUtbikxxN81tY0Nx93PSEPYJr8xxmQA/rTvmkz4t7EJP0MNQ00NzB4rXblxsGBb7hdtLZkjMgVBxe


### Steps to Reproduce

1. Install game
2. Start game
3. End turn
4. Crash.

### Screenshots

_No response_

### Link to save file

_No response_

### Operating System

Android

### Additional Information

_No response_

--- a/core/src/com/unciv/logic/files/UncivFiles.kt
+++ b/core/src/com/unciv/logic/files/UncivFiles.kt
@@ -82,7 +82,8 @@ class UncivFiles(
                 externalFile.exists() && !localFile.exists() || // external file is only valid choice
                 preferExternalStorage && (externalFile.exists() || !localFile.exists()) // unless local file is only valid choice, choose external
                 ) ) {
-            externalFile.mkdirs()
+            if (externalFile.isDirectory) externalFile.deleteDirectory() // fix for #13113
+            externalFile.parent().mkdirs()
             externalFile
         } else {
             localFile

